src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraphCheck.java

```diff
@@ -2,23 +2,33 @@
 
 /**
  * Some Javadoc.
- * <p> //warn
- * /^ WARN/   Some Javadoc.<p> //warn
- *
+ * <p>
+ * // 2 violations above:
```

Comment:
Please move all such marker comments outside of javadoc comment at all.
```
// Violation 2 lines below....
/**
 * Some Javadoc.
 * <p> some text
 *    Some Javadoc.<p>
 */
```
```

Reply:
Not done

Reply:
done